test/integration/scheduler/priorities_test.go

```diff
@@ -361,3 +362,105 @@ func TestEvenPodsSpreadPriority(t *testing.T) {
 		})
 	}
 }
+
+// TestDefaultPodTopologySpreadScore verifies that the PodTopologySpread Score plugin
+// with the system default spreading spreads Pods belonging to a Service.
+// The setup has 500 nodes over 3 zones.
+func TestDefaultPodTopologySpreadScore(t *testing.T) {
+	testCtx := initTestSchedulerForPriorityTest(t, podtopologyspread.Name)
+	t.Cleanup(func() {
+		testutils.CleanupTest(t, testCtx)
+	})
+	cs := testCtx.ClientSet
+	ns := testCtx.NS.Name
+
+	var nodes []*v1.Node
+	for i := 0; i < 500; i++ {
+		nodeName := fmt.Sprintf("node-%d", i)
+		zone := fmt.Sprintf("zone-%d", i%3)
+		node, err := createNode(cs, st.MakeNode().Name(nodeName).Label(v1.LabelHostname, nodeName).Label(v1.LabelZoneFailureDomainStable, zone).Obj())
```

Comment:
Are you aware of any trick to create the nodes in batch? It seems we will have to issue 500 API calls :)

Reply:
I don't think there are batch API calls. But this is integration, it actually runs pretty fast. Let me lower to 300 just in case.
Btw, I just want to be sure I'm reasonably past the 100 window size.